src/net.cpp

```diff
@@ -2813,35 +2811,34 @@ bool CConnman::NodeFullyConnected(const CNode* pnode)
 
 void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
 {
-    size_t nMessageSize = msg.data.size();
-    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n",  SanitizeString(msg.m_type), nMessageSize, pnode->GetId());
+    size_t message_size = msg.data.size();
+    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n", SanitizeString(msg.m_type), message_size, pnode->GetId());
 
     // make sure we use the appropriate network transport format
-    std::vector<unsigned char> serializedHeader;
-    pnode->m_serializer->prepareForTransport(msg, serializedHeader);
-    size_t nTotalSize = nMessageSize + serializedHeader.size();
+    std::vector<unsigned char> serialized_header;
+    pnode->m_serializer->prepareForTransport(msg, serialized_header);
+    size_t total_size = message_size + serialized_header.size();
 
-    size_t nBytesSent = 0;
+    bool optimistic_send{false};
     {
         LOCK(pnode->cs_vSend);
-        bool optimisticSend(pnode->vSendMsg.empty());
+        optimistic_send = pnode->vSendMsg.empty();
 
         //log total amount of bytes per message type
-        pnode->mapSendBytesPerMsgCmd[msg.m_type] += nTotalSize;
-        pnode->nSendSize += nTotalSize;
+        pnode->mapSendBytesPerMsgCmd[msg.m_type] += total_size;
+        pnode->nSendSize += total_size;
 
         if (pnode->nSendSize > nSendBufferMaxSize)
             pnode->fPauseSend = true;
-        pnode->vSendMsg.push_back(std::move(serializedHeader));
-        if (nMessageSize)
+        pnode->vSendMsg.push_back(std::move(serialized_header));
+        if (message_size)
```

Comment:
Since you're cleaning everything up here, this if block (and the one above) should have braces added.

Reply:
Good eye, thanks